sample-app/notification-sample-code/nodejs/README.md

```diff
@@ -0,0 +1,121 @@
+# Notification Sample Code
+
+This sample code makes use of the Create Subscription endpoint to create a subscription for ringing events on a range of endpoints defined by the user. This sample code is expected to output any received webhooks to the console
+
+## Pre-requisites
+
+[Node.js](https://nodejs.org/en/) - Install node.js
```

Comment:
We should probably add having a cloudlink account and obtaining a cliend Id as pre- requisites.

@buddrenaud, maybe we can link or at least tell where they can read your get starting doc?